src/configs/index.js

```diff
@@ -253,3 +258,14 @@ export const CONFIRM_ERROR = [
     value: 'Oops! Something went wrong.'
   }
 ];
+
+export const ERROR_PAGE_MESSAGE = [
```

Comment:
Please, move it from config to the `translations` folder, in a separate file. Look how other translations are implemented